.github/renovate.json

```
@@ -21,5 +21,15 @@
       ],
       "matchUpdateTypes": ["minor", "patch"]
     }
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["proxy-actions/codecov/action.yml$"],
+      "matchStrings": [
+        "version:[\\s\\S]*?default: (?<currentValue>\\d+\\.\\d+\\.\\d+)"
```

Fixes the renovate config after changing the default value to `v11.2.6`

```suggestion
        "version:[\\s\\S]*?default: (?<currentValue>v\\d+\\.\\d+\\.\\d+)"
```